pkg/logs/logs.go

```diff
@@ -49,8 +48,6 @@ const (
 	TraceLevel        = 5
 )
 
-var logFlushFreq = flag.Duration("log-flush-frequency", 5*time.Second, "Maximum number of seconds between log flushes")
```

Comment:
I propose that we re-add this flag. We will deprecate it in https://github.com/cert-manager/cert-manager/pull/5879 and remove it (maybe, one day...)